<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>507</height>
   </rect>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>711</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Status Bar</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_Update">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>380</y>
        <width>211</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Read from Home Screen App</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_Status">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>681</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_IconExample">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>380</y>
        <width>141</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Icon-Example</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Popup">
     <attribute name="title">
      <string>Popup</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_Send">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>380</y>
        <width>99</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="plainTextEdit_Popup">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>681</width>
        <height>361</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_PopupExample">
      <property name="geometry">
       <rect>
        <x>428</x>
        <y>380</y>
        <width>171</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>&quot;Hello World&quot; Example</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_comboBoxExample">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>380</y>
        <width>191</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>ComboBox Example</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_DayNightMode">
     <attribute name="title">
      <string>Day/Night Mode</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_DayNightMode">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>161</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Day/Night Mode</string>
      </property>
      <widget class="QRadioButton" name="radioButton_NightMode">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Night Mode</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_DayMode">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>117</width>
         <height>22</height>
        </rect>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="text">
        <string>Day Mode</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_Proximity">
     <attribute name="title">
      <string>Proximity</string>
     </attribute>
     <widget class="QLabel" name="label_explanation">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>301</width>
        <height>231</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Wikipedia:&lt;/p&gt;&lt;p&gt;A proximity sensor is a sensor able to detect the presence of nearby objects without any physical contact.&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;This info can be used to detect the hand of the user that wants to use the touch screen. So the HMI can change its look to provide for example larger push buttons.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="checkBox_ObjectDetected">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>300</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>object detected</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_FutureFeatures">
     <attribute name="title">
      <string>Future features</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_Right">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>160</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Right</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_OK">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>160</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Left">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Left</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Down">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>240</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Down</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Up">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>80</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Up</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>181</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>under development
not functional right now</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Cancel">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>240</y>
        <width>71</width>
        <height>71</height>
       </rect>
      </property>
      <property name="focusPolicy">
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text">
       <string>Cancel/
Back?</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_Exit">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>470</y>
      <width>99</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
